hack/update-helm-docs.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+HELM_DOCS_BIN="${KUBE_ROOT}/bin/helm-docs"
+
+$HELM_DOCS_BIN ${KUBE_ROOT}/deploy/charts/google-cas-issuer
```

Comment:
We should add this to cert-manager as well, I think the contents of https://github.com/cert-manager/cert-manager/blob/master/deploy/charts/cert-manager/README.template.md are all handwritten